CruzHacks/cruzhacks-2021-website
# added status icons and feedback variable

Problem
=======
As a user,
I would like to see graphical icons when I submit my email
to give me a better understanding of how my request was handled!

Solution
========
Added `react-spinners` and used https://www.icongr.am/ to source success/failure icons for the email subscription component.

Change summary:
---------------
* added status icons and feedback variable 9b93638

Steps to Verify:
----------------
1. Checkout branch using `git checkout enhancement/email-icons`
2. Start local development server using `yarn start`
4. Submit email in form to verify that appropriate response message and loading/static icons are returned
